packages/client-search/src/types/SearchResponse.ts

```diff
@@ -224,6 +225,10 @@ export type SearchResponse<TObject = {}> = {
      * as a JSON object with one field per parameter.
      */
     params?: Record<string, any>;
+
+    redirect?: {
```

Comment:
if we have a link to the public doc, it would make sense to me to add it here

Reply:
This parameter is not public and is not communicated to customers.

Reply:
@shortcuts added a comment for internal use